2-projects/1-todo-list/script.js

```diff
@@ -9,17 +31,86 @@ let todos = [
   { task: "Wash the dishes", completed: false },
   { task: "Do the shopping", completed: false },
 ];
-
 populateTodoList(todos);
 
-// This function will take the value of the input field and add it as a new todo to the bottom of the todo list. These new todos will need the completed and delete buttons adding like normal.
+// Access to the input
+const input = document.querySelector("#todoInput"),
```

Comment:
i didnt get the syntax. how did you manage to define variables without using const or let keyword?